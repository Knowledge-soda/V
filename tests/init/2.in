+(a, b)
f(x, y) := *(x, y)
