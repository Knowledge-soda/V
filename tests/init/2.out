_start:
    NAME(id) BEGIN NAME(+) BEGIN NAME(a) SEP NAME(b) END END
f:
    BEGIN NAME(x) SEP NAME(y) END : NAME(*) BEGIN NAME(x) SEP NAME(y) END
